config.json

```diff
@@ -1361,6 +1361,16 @@
         "bitwise_operations",
         "transforming"
       ]
+    },
+    {
+      "slug": "high-scores",
+      "uuid": "19b41c4a-1466-5c19-b547-40284189fd18",
+      "core": false,
+      "unlocked_by": "acronym",
+      "difficulty": 2,
+      "topics": [
```

Comment:
Using the same `core`, `unlocked_by`, `difficulty` and `topics` as [the Ruby track](https://github.com/exercism/ruby/blob/master/config.json#L225). Do these settings make sense for this track?

Reply:
Just saw the [build](https://travis-ci.org/exercism/javascript/builds/447392671?utm_source=github_status&utm_medium=notification). 😅 Apparently not  

> The command "bin/fetch-configlet" exited with 0.
> 0.19s$ bin/configlet lint .
> -> The exercise 'high-scores' is being unlocked by a non-core exercise. Non-core exercises can only be unlocked by core exercises.

Anyone have any suggestions? Would anyone object to `unlocked_by` being set to null?

Reply:
@PakkuDon I'm thinking: have it be unlocked by `pangram`?  Since that's the one that unlocks `acronym`?